### ISSUE TITLE
Merge back to strands-project

### ISSUE THREAD

**Issue by hawesie** [bfalacerda/strands_executive#18]
Can we get everything merged back to the released branch this week? Does prism-robots need to be updated too?

**Comment by bfalacerda** [comment]
maybe we should create an indigo-devel branch to avoid conflicts with AAF deployment, and because hydro-release is completely different from our other branch names?

It does require prism-robots re-release, do you remember where the instructions are to make a .deb file to send Marc?

**Comment by hawesie** [comment]
Agreed on indigo-devel, with this becoming the main released branch post-AAF deployment. I will make my next PR there.

**Comment by hawesie** [comment]
For the deb stuff, no idea, sorry!
